Simplenote/src/main/java/com/automattic/simplenote/PreferencesActivity.java

```diff
@@ -24,14 +24,8 @@
     private PasscodePreferenceFragmentCompat mPasscodePreferenceFragment;
```

Comment:
We can remove the `import android.content.Intent;`, `import androidx.core.app.NavUtils;`, and `import com.automattic.simplenote.utils.ThemeUtils;` statements since they are no longer used.